ui/ducks/swaps/swaps.test.js

```diff
@@ -401,64 +401,6 @@ describe('Ducks - Swaps', () => {
     });
   });
 
-  describe('getSmartTransactionsEnabled', () => {
```

Comment:
Some code and UI were moved outside of Swaps, so that's why you can notice a bit of a cleanup there.